src/server/transaction.cc

```diff
@@ -994,6 +962,11 @@ void Transaction::ExecuteAsync() {
   });
 }
 
+void Transaction::FinishHop() {
+  boost::intrusive_ptr<Transaction> guard(this);  // Keep alive until Dec() fully finishes
```

Comment:
I added a suggestion to add hop index to the debug info.
you think it's not useful?

Reply:
I asked where do you want to store it 😆 Shard stats or separate index that is incremented just after notifying?

Reply:
If you want to count it per tx I have to bach BlockingCounter::Dec to return bool /* last */

Reply:
i did not mean to count it :)
just for debugging when we have issues. to understand at what hope the crash happens for example